ctakesclient/typesystem.py

```diff
@@ -166,7 +166,7 @@ def from_json(self, source: dict):
         self.conceptAttributes = []
 
         # sort list of concepts ensuring same ordering
-        unsorted = list(UmlsConcept(c) for c in source.get('conceptAttributes'))
```

Comment:
OK. Curious when is conceptAttributes missing? Is this only when using "Identified" custom dictionary annotations that are from the 3 column BSV? Either way, seems like a good check.

Reply:
Ah yeah this was an unrelated change I just had sitting on my laptop from earlier testing. I had manually constructed a MatchText I think in a test file and hit an exception because it didn't have a conceptAttributes defined. Easy enough to fix in the test file (I just added the field). But I had also removed the requirement here and am just now committing it.

It should always be defined anyway, but I figured no harm in being a bit graceful when its not.